v2/hack/certify/certify.sh

```diff
@@ -0,0 +1,164 @@
+
+# Create the kubeconfig used by the certification pipeline
+oc delete secret kubeconfig --ignore-not-found
+oc create secret generic kubeconfig --from-file=kubeconfig=$KUBECONFIG
+
+# Import redhat catalogs
+oc import-image certified-operator-index \
+  --from=registry.redhat.io/redhat/certified-operator-index \
+  --reference-policy local \
+  --scheduled \
+  --confirm \
+  --all
+
+oc import-image redhat-marketplace-index \
+  --from=registry.redhat.io/redhat/redhat-marketplace-index \
+  --reference-policy local \
+  --scheduled \
+  --confirm \
+  --all
+
+CWD=$(pwd)
+TMP_DIR=$(mktemp -d 2>/dev/null || mktemp -d -t 'cptmpdir')
+
+# Install the Certification Pipeline
+cd $TMP_DIR
+git clone https://github.com/redhat-openshift-ecosystem/operator-pipelines
+cd operator-pipelines
+oc apply -R -f ansible/roles/operator-pipeline/templates/openshift/pipelines
+oc apply -R -f ansible/roles/operator-pipeline/templates/openshift/tasks
+cd $CWD
+
+oc apply -f https://raw.githubusercontent.com/tonytcampbell/operator-pipelines/preflight-fixes/ansible/roles/operator-pipeline/templates/openshift/tasks/preflight.yml
+
+
+# Add bundle to the fork
+cd $TMP_DIR
+git clone git@github.com:redhat-marketplace/certified-operators-preprod.git
+
+cd $CWD
+cd ../../
+
+rm -rf $TMP_DIR/certified-operators-preprod/operators/redhat-marketplace-operator/$VERSION
+mkdir -p $TMP_DIR/certified-operators-preprod/operators/redhat-marketplace-operator/$VERSION
+cp -r bundle/manifests $TMP_DIR/certified-operators-preprod/operators/redhat-marketplace-operator/$VERSION/
+cp -r bundle/metadata $TMP_DIR/certified-operators-preprod/operators/redhat-marketplace-operator/$VERSION/
+
+# The operator service account should be ommited in the bundle
+# It will fail certification
+# The service account will be created by OLM
+# kustomize questionable capability to remove the service account
+rm -Rf  $TMP_DIR/certified-operators-preprod/operators/redhat-marketplace-operator/$VERSION/manifests/redhat-marketplace-operator_v1_serviceaccount.yaml
+
+echo "organization: redhat-marketplace" > $TMP_DIR/certified-operators-preprod/config.yaml
+echo "cert_project_id: 5f68c9457115dbd1183ccab6" > $TMP_DIR/certified-operators-preprod/operators/redhat-marketplace-operator/ci.yaml
+
+cd $TMP_DIR/certified-operators-preprod
+git add --all
+git commit -m $VERSION
+git push
+
+
+# Run the Pipeline
+
+cd $TMP_DIR/operator-pipelines
+curl https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.19.1/tkn-linux-amd64-0.19.1.tar.gz | tar -xz 
+
+GIT_REPO_URL=https://github.com/redhat-marketplace/certified-operators-preprod.git
+BUNDLE_PATH=operators/redhat-marketplace-operator/$VERSION
+
+./tkn pipeline start operator-ci-pipeline \
+  --param git_repo_url=$GIT_REPO_URL \
+  --param git_branch=stage \
+  --param bundle_path=$BUNDLE_PATH \
+  --param env=stage \
+  --workspace name=pipeline,volumeClaimTemplateFile=templates/workspace-template.yml \
```

Comment:
Does there need to exist a template/workspace-template.yml somewhere? Or is this in the other repo?

Reply:
At this point in the script, we are in the operator-pipelines dir, so the file is

https://github.com/redhat-openshift-ecosystem/operator-pipelines/blob/main/templates/workspace-template.yml